cli_tools/gce_vm_image_import/cli/cli.go

```diff
@@ -92,6 +93,17 @@ func Main(args []string, toolLogger logging.ToolLogger, workflowDir string) erro
 	return nil
 }
 
+func userFriendlyError(err error, importArgs importer.ImportArguments) error {
+	if strings.Contains(err.Error(), "constraints/compute.vmExternalIpAccess") {
```

Comment:
probably want a nil check since you'll be dereferencing `err`.

Reply:
Yep, missed that. Done